docs/architecture/microservices/net-core-net-framework-containers/net-framework-container-scenarios.md

```diff
@@ -15,31 +15,29 @@ In most cases for this scenario, you will not need to migrate your existing appl
 
 - Workflow-related services. Windows Workflow Foundation (WF), Workflow Services (WCF + WF in a single service), and WCF Data Services (formerly known as ADO.NET Data Services) are only available on .NET Framework. There are currently no plans to bring them to .NET Core.
 
-In addition to the technologies listed in the official [.NET Core roadmap](https://github.com/dotnet/aspnetcore/wiki/Roadmap), other features might be ported to .NET Core. For a full list, look at the items tagged as [port-to-core](https://github.com/dotnet/corefx/issues?q=is%3Aopen+is%3Aissue+label%3Aport-to-core) on the CoreFX GitHub site. Note that this list does not represent a commitment from Microsoft to bring those components to .NET Core — the items simply capture requests from the community. If you care about any of the components listed above, consider participating in the discussions on GitHub so that your voice can be heard. And if you think something is missing, please [file a new issue in the Runtime repository](https://github.com/dotnet/runtime/issues/new).
+In addition to the technologies listed in the official [.NET Core roadmap](https://github.com/dotnet/core/blob/master/roadmap.md), other features might be ported to .NET Core or the new [unified .NET platform](https://devblogs.microsoft.com/dotnet/introducing-net-5/). You might consider participating in the discussions on GitHub so that your voice can be heard. And if you think something is missing, please [file a new issue in the Runtime repository](https://github.com/dotnet/runtime/issues/new).
 
-Even though .NET Core 3 (at the time of this writing this is in the works) will include support for a lot of existing .NET Framework APIs, these are desktop oriented so, currently, they are of no use in the container world.
+## Using a platform or API that doesn't support .NET Core
 
-## Using a platform or API that does not support .NET Core
-
-Some Microsoft or third-party platforms do not support .NET Core. For example, some Azure services provide an SDK that is not yet available for consumption on .NET Core. This is temporary, because all Azure services will eventually use .NET Core. For example, the [Azure DocumentDB SDK for .NET Core](https://www.nuget.org/packages/Microsoft.Azure.DocumentDB.Core) was released as a preview on November 16, 2016, but it is now generally available (GA) as a stable version.
+Some Microsoft and third-party platforms don't support .NET Core. For example, some Azure services provide an SDK that isn't yet available for consumption on .NET Core. This is temporary, because all Azure services will eventually use .NET Core/Standard. You can see the available Azure SDKs in the [Azure SDK Latest Releases](https://azure.github.io/azure-sdk/releases/latest/index.html) page.
```

Comment:
Yep, perhaps that was the idea back then but I think it's fair to change this to something not so absolute, because anything can happen, like considering some tech is no longer relevant.

Reply:
Changed.